packages/flodash/lib/modules/string/to_upper.dart

```diff
@@ -0,0 +1,4 @@
+///Converts string, as a whole, to upper cas
```

Comment:
There looks to be a spelling mistake in the comment here, shall we add in a fix for it ?